src/common/utils/kaltura-params.ts

```diff
@@ -15,6 +15,12 @@ const SIGNATURE = 'signature=';
 const SEEK_FROM = 'seekFrom=';
 const CLIP_TO = 'clipTo=';
 
+declare global {
+  interface Window {
```

Comment:
this doesn't conflict with other definitions of Window ?
do we have another definition for Window where we can put this ?